server/lib/src/constants/schema.rs

```diff
@@ -14,7 +14,6 @@ pub static ref SCHEMA_ATTR_DISPLAYNAME: SchemaAttribute = SchemaAttribute {
     uuid: UUID_SCHEMA_ATTR_DISPLAYNAME,
     name: Attribute::DisplayName.into(),
     description: "The publicly visible display name of this person".to_string(),
-
```

Comment:
The newlines are intentional in this and the rest of the objects, to break up the identifier data and the policy data, please return them.

Reply:
Did my best. Please go on.